### DISCUSSION TITLE
Welcome to online-complier Discussions!

### DISCUSSION THREAD

**Discussion by saswatsamal** [hackclubiter/onEditor#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LoopGlitch26** [comment]
Hello World!

I'm **Bravish Ghosh**, a passionate enthusiast for innovation and development. I love to research and code for the sake of value addition to society. I'm a part of this wonderful student community, **SOA CODE ROOM**. I am a core member and my focus is to help the moderation team in supporting the community development, as well as guide and collaborate with various members.

Tech Stack Support: Python, Flask, HTML/CSS, JavaScript, etc.

**Reply by saswatsamal** [reply]
I'll love to work with you! 🤓
